mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/blob/BlobStoreDeletedMessageVault.java

```diff
@@ -66,21 +65,24 @@
 
     private final MetricFactory metricFactory;
     private final DeletedMessageMetadataVault messageMetadataVault;
-    private final DeduplicatingBlobStore blobStore;
+    private final BlobStore blobStore;
```

Comment:
I think deduplication is expected 

Cf `MemoryDeletedMessageVaultIntegrationTest>DeletedMessageVaultIntegrationTest.vaultDeleteShouldDeleteAllMessagesHavingSameBlobContent`

That's what I had been told when I came up with https://github.com/linagora/james-project/pull/3028

Could you at the very least move this commit in a separated pull request?